tests/generators/OperationGenerator.ts

```diff
@@ -338,7 +332,7 @@ export default class OperationGenerator {
     const encodedDeltaString = Encoder.encode(deltaJsonString);
 
     const signedDataPayloadObject = {
-      update_reveal_value: updateRevealValue,
+      update_key: updateKey,
       delta_hash: deltaHash
     };
     const signedData = await OperationGenerator.signUsingEs256k(signedDataPayloadObject, signingPrivateKey, signingKeyId);
```

Comment:
Consider just changing method signature to just not allow key ID for `OperationGenerator.signUsingEs256k()`